app/assets/javascripts/find-data.js

```diff
@@ -0,0 +1,13 @@
+function sortDatasets() {
+    document.location.href = newLocation()
+
+    function newLocation() {
```

Comment:
I don't think that creating a newLocation function is necessary. It brings the risk of things breaking is the declaration ever changes to ` var newLocation = function...` and stop the hoisting of the function. Better to just write `document.location.href = function () {...`